.github/workflows/android.yml

```diff
@@ -0,0 +1,27 @@
+name: cross compile with android-ndk
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  cross-compile-for-android:
+    strategy:
+      fail-fast: false
+      matrix:
+        target-abi: [armeabi-v7a, arm64-v8a, x86, x86_64]
+
+    runs-on: macos-10.15
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - run:  brew install tree file
+
+      - run:  patch -p1 < android.patch
```

Comment:
Why not just apply the patch and persist that change?

Reply:
Do you mean why not I file a pull request to apply the android.patch?

Reply:
Yes, let's just apply the patch and commit these changes instead of having to maintain the patch file

Reply:
![28](https://user-images.githubusercontent.com/3040852/151808740-4dd24b46-a448-416f-a0ba-a4807ebee7ee.PNG)

I have said in https://github.com/dbrgn/tealdeer/issues/260

have you read the patch? do you accept this patch file. I'm fairly new to Rust language. I have no ablity to write perfect code in Rust now.

Reply:
The patch seems to go in the right direction, not a lot of Rust is required (and we can help you / tell you what to do to fix errors if they come up). Just run `patch -p1 < android.patch`, remove `android.patch` and push all of these changes here (if you need git help, feel free to reach out too) :)